### ISSUE TITLE
Upload fails on ansible box

### ISSUE THREAD

**Issue by mattl** [w3c/wptdashboard#518]
CommandException: 25568 files/objects could not be copied/removed.
Traceback (most recent call last):
  File "./run/run.py", line 599, in <module>
    main(args.platform_id, platform, args, config)
  File "./run/run.py", line 303, in main
    return_code = subprocess.check_call(command, cwd=config['build_path'])
  File "/usr/lib/python2.7/subprocess.py", line 541, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['gsutil', '-m', '-h', 'Content-Encoding:gzip', 'rsync', '-r', u'72a61489a6', u'gs://wptd/72a61489a6']' returned non-zero exit status 1

**Comment by mattl** [comment]
Running it manually now.

**Comment by mattl** [comment]
Exits with an error per file: ServiceException: 401 Anonymous caller does not have storage.objects.create access to wptd/72a61489a6/chrome-64.0-linux/xhr/xmlhttprequest-sync-block-defer-scripts.html.

Then finally with:

ServiceException: 401 Anonymous caller does not have storage.objects.create access to bucket wptd.
CommandException: 25568 files/objects could not be copied/removed.

**Comment by jugglinmike** [comment]
As part of the the work I did to make the Ansible scripts work, I had to re-implement authentication. This was because our existing solution was tied to a specific Google Cloud user account, and that isn't a tenable long-term solution. My initial guess is that the credentials have expired.

One piece of data that will help confirm this is if another system exhibits the same failure. I will re-deploy the Buildbot implementation and check from there.

Given this issue blocks progress on the application of these deployment scripts, and given that we expect to move to a new set of scripts in the near future, I think we should de-prioritize our efforts to use these scripts.
